ingestions/ingest_csvs.py

```diff
@@ -27,18 +30,79 @@
 PRIORITY = 1
 
 # initialize user credentials and the OOINet base URL
-BASE_URL = 'https://ooinet.oceanobservatories.org'
+# BASE_URL = 'https://ooinet.oceanobservatories.org'
+BASE_URL = 'https://ooinet-west.oceanobservatories.org'
+DEPLOY_URL = '12587/events/deployment/inv/'
 credentials = netrc.netrc()
 API_KEY, USERNAME, API_TOKEN = credentials.authenticators('ooinet.oceanobservatories.org')
```

Comment:
I had to change this to ooinet-west.oceanobservatories.org

Reply:
Changed the status code to 'create' to enable dataframe creation and properly parse the response. Changed the netrc host name to ooinet-west.oceanobservatories.org and created an entry for that host in my .netrc file. So long as the host name corresponds to an entry in the .netrc file, this will work. I've now tested this both with and without the `-y` option and it all works as expected.